### ISSUE TITLE
Find and replace text in the footer

### ISSUE THREAD

**Issue by devxplorer** [python-openxml/python-docx#203]
Hello everyone, i cant find a way to find and replace certain text in the footer.

Can anyone please show me how to do it?

**Comment by adi-** [comment]
Would like to know too

**Comment by blindmelon** [comment]
Did either of you work this out?

**Comment by devxplorer** [comment]
No, actually i dont. Looks like python-docx have no native functionality to perform this.
Had to use this library http://docxfactory.com/
